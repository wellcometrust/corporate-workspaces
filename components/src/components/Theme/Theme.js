// @flow
import * as React from 'react';
import { ThemeProvider } from 'styled-components';

import theme from '../../styles/theme';

type Props = {|
  children: React.Node
|};

const Theme = ({ children }: Props) => (
  <ThemeProvider theme={theme}>{children}</ThemeProvider>
);

export default Theme;
