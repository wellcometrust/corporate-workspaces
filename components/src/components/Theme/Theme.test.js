import React from 'react';
import { shallow } from 'enzyme';

import Theme from './Theme';

describe('<Theme />', () => {
  const output = shallow(<Theme />);

  it('renders the component', () => {
    expect(output).toMatchSnapshot();
  });
});
