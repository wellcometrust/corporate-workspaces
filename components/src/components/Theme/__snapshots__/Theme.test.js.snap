// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Theme /> renders the component 1`] = `ShallowWrapper {}`;
